learn/index.md

```diff
@@ -1,5 +1,9 @@
 ---
-title: Learn
-description: Tanssi docs....
+title: Learn About Tanssi
+description: Learn all about Tanssi, how to build your appchain with the rust language based substrate framework and how easy is to deploy your Appchain in the polkadot web3 ecosystem
```

Comment:
Too long. Substrate, Polkadot, and Web3 should be capitalized and don't forget to add punctuation at the end of the sentence!

Also depending on what you decide (based off other comment I made on the templates page), Appchain should be capitalized